pkg/api/v1alpha1/cluster.go

```diff
@@ -50,6 +53,13 @@ func NewClusterGenerate(clusterName string, opts ...ClusterGenerateOpt) *Cluster
 				CNI: Cilium,
 			},
 		},
+		Status: ClusterStatus{
```

Comment:
I would vote to not generate this here and just set in the cli during create/upgrade
I don't think we should concern the user with this

Reply:
Sure, makes sense. If we want to allow the user the configure the namespace for this resource in the future, we will have to expose this.